index.html

```diff
@@ -316,5 +328,123 @@ <h3 class="about-me__card__title">Skills</h3>
         </div>
       </div>
     </section>
+
+    <!-------CONTACT SECTION ------->
+    <section id="contact">
+      <div class="contact">
+        <div class="contact__container">
+          <h2>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
+
+          <form
+            class="contact__form"
+            action="https://formspree.io/f/xknaqawg"
+            method="POST"
+          >
+            <ul>
+              <li class="contact__form__element full-name">
+                <label class="label" for="full-name">Full name:</label>
+                <input
+                  type="text"
+                  id="full-name"
+                  name="full-name"
+                  placeholder="Full name"
+                  maxlength="30"
+                  required
+                />
+              </li>
+
+              <li class="contact__form__element name">
+                <label class="label" for="firstname">First name:</label>
+                <input
+                  type="text"
+                  id="firstname"
+                  name="firstname"
+                  placeholder="First name"
+                  maxlength="30"
+                  required
+                />
+              </li>
+
+              <li class="contact__form__element lastname">
+                <label class="label" for="lastname">Last name:</label>
+                <input
+                  type="text"
+                  id="lastname"
+                  name="lastname"
+                  placeholder="Last Name"
+                  maxlength="30"
+                  required
+                />
+              </li>
+
+              <li class="contact__form__element email">
+                <label class="label" for="email">Email address:</label>
+                <input
+                  type="email"
+                  id="email"
+                  name="email"
+                  placeholder="Email address"
+                  required
+                />
+              </li>
+
+              <li class="contact__form__element message">
+                <label class="label" for="message">Your message:</label>
+                <textarea
+                  id="message"
+                  name="message"
+                  placeholder="Hey I'd like to connect to..."
+                  maxlength="500"
+                  required
+                ></textarea>
+              </li>
+            </ul>
```

Comment:
- [ ] As mentioned in the previous review and for the project requirements, kindly ensure that when a user **clicks** the submit button, the form message is correctly sent for the project requirements. Right now, it shows the below err![Screenshot from 2023-05-03 02-36-20](https://user-images.githubusercontent.com/83042742/235813381-76126658-1803-40d7-914e-278f93fd28fa.png)

**IMPORTANT:** **Kindly make sure you use the same pull request and not create a new one**

Reply:
Yes, you are right. I'm solving it! Thanks!

Reply:
@tresorsawasawa I found the mistake; so if you see the template, both forms, desktop and mobile, are different, so in mobile version requires only 3 fields "Full name", " email," and the "message". And in the desktop version requires 4 "First name", "Last name," "email," and "message." So, for the mobile version, it's needed to hide (display: none) the fields  "First name"& "Last name" to fit with the template, and of course, in the mobile version, it's needed to hide the "Full name" field.

So, I can add "required" to any of these fields because they are different in both templates. On the contrary, the error still appears. Please consider this and respond to my comment because this is my last code review. As you have seen, I have put all my effort into this, and it would be a pity to fail because of an unfair issue...

Reply:
The other issue is, I can add a limit of 30 characters to the "Full name" feature, because there exist full names over that amount, for example my full name: Sergio Andres Usma Estupiñan has 28 characters, and even exist more longer full names. So, please consider this too...

Reply:
@tresorsawasawa please confirm you understand the case before to re-send the code review. I have made the changes, but this is something related to the template, not to me...

Reply:
@tresorsawasawa this is the last commit, please check

![image](https://user-images.githubusercontent.com/126805818/235818041-cb7b0597-7791-4d22-a2c9-4cf35125a8c6.png)

Reply:
Hello @sergio-usma 
I can understand your concern related to different input fields for mobile and desktop versions. Please focus on the current milestone only, which is related to mobile version. You will get the requirements for the desktop version and how you will deal with different input fields later on. I hope this helps. Let me know if you have any concerns.
I wish you success. Good Luck! 🥇

Reply:
Ok, but this was my last review... So, I don't know what to do now....

Reply:
You have 3 extra reviews. You can use one of them

Reply:
Ok, perfect! Thanks!